mlir/test/Target/LLVMIR/nsw_nuw.mlir

```diff
@@ -0,0 +1,14 @@
+// RUN: mlir-translate -mlir-to-llvmir %s | FileCheck %s
+
+// CHECK-LABEL: define void @intflags_func
+llvm.func @intflags_func(%arg0: i64, %arg1: i64) {
+  // CHECK: %{{.*}} = add nsw i64 %{{.*}}, %{{.*}}
+  %0 = llvm.add %arg0, %arg1 flags <nsw> : i64
```

Comment:
I swapped to "overflow", but dropping the space wasn't trivial. Would you like me to pursue it?

Reply:
Arith does the following for the fastmath flags (`fastmath` `` $fastmath^)?`. I suspect the double ticks remove the space? If you can give it a shot that would be nice otherwise the current solution sounds fine as well.